maven-release-plugin/src/main/java/org/apache/maven/plugins/release/BranchReleaseMojo.java

```diff
@@ -199,12 +199,12 @@
     private String projectVersionPolicyId;
 
     /**
-     * The role-hint for the NamingPolicy implementation used to calculate the project branch and tag names.
+     * The role-hint for the NamingPolicy implementation used to calculate the project names.
      *
      * @since 3.0.0
      */
-    @Parameter( defaultValue = "default", property = "projectNamingPolicyId" )
-    private String projectNamingPolicyId;
+    @Parameter( defaultValue = "default-branch", property = "projectBranchNamingPolicyId" )
```

Comment:
If the default value is "default", then it selects the same policy as the default tag naming policy because there is only a single map with policies. That is the whole point of this change, as the default policy is *different* for tags (there is uses <artifact>-<version> if no tag was given) and branches (there it provides *no* default therefore a branch request *fails* if no name is given). As you insisted to moving this naming strategy into the DefaultNamingStrategy and refused any approach to separate those out, what is your suggestion to continue?

I am at a loss here. I am going back and forth through these changes and every time, you come back saying "this needs to be different, not this way". There are *two* default policies and you block any approach to acknowledge this fact.